README.md

```diff
@@ -122,6 +122,11 @@ has been authenticated through the token. This is then accessible in the route.
 *Future updates of this filter should add more information from the token into the context.*
 
 
+## Access token claims in Request object
+
+When the filter accepts the request, it sets the `request.foil_claims` context local variable for that request with all
```

Comment:
why namespace this? Why not juse use `claims`? So, `request.claims`. Will be more intuitive and look nicer IMO.

Reply:
sure i ll change that, just wanted to be safe. it is more likely that the request object will contain `claims` in the future than `foil_claims`.  Probably not worth the ugnliness though.